src/components/calcite-block/calcite-block.tsx

```diff
@@ -135,13 +135,13 @@ export class CalciteBlock {
       <slot name={SLOTS.icon} />
     ) : (
       <calcite-icon
-        icon={icon}
-        scale="m"
         class={{
           [CSS.statusIcon]: true,
           [CSS.valid]: status == "valid",
           [CSS.invalid]: status == "invalid"
         }}
+        icon={icon}
```

Comment:
Optional: you can leave this as it was to keep the commit/PR focused and we can run linting across all components to avoid inconsistencies across PRs. Up to you.